tests/lib/pkgdb.sh

```diff
@@ -518,6 +525,10 @@ distro_install_build_snapd(){
             fi
         fi
 
+        if ! systemctl is-active "$mount_unit"; then
```

Comment:
Can you add a comment that explains why the unit is started. I think it's because not all packages will start it but perhaps it's not the full story.